### PR TITLE
Cerb benefit

### PR DESCRIPTION
**Pull request by MikeBungay**
implemeneted rent help and tests

### CODE REVIEW

File: openfisca_canada/tests/benefitfinder/rent.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  input:
+    persons:
+      person_needs_rent_help:
+        rent__has_need_for_rent_help: true
+  output:
+    rent__is_eligible:
+      - true   
+ 
+#Does not Have need for rent help
+- name: "individual - has a riff"
```
Comment (MarcoGoC) [inline openfisca_canada/tests/benefitfinder/rent.yaml]: has a riff ??

**Review by shewood — APPROVED** [review]
Looks good to me.

**Review by MarcoGoC — APPROVED** [review]
except for the name in the test all good

File: openfisca_canada/tests/benefitfinder/rent.yaml (hunk 1 above)
Comment (MikeBungay) [reply openfisca_canada/tests/benefitfinder/rent.yaml]: fixed

**Pull request merged**